### PR TITLE
fixed: current user team ids fetch issue in azure-devops plugin

### PR DESCRIPTION
**Pull request by deepan10**
Signed-off-by: Deepankumar Loganathan <deepan0433@gmail.com>

## Hey, I just made a Pull Request!

This PullRequest will fix issue #15760.

Now the filters  AssignedToCurrentUsersTeams & CreatedByCurrentUsersTeams are working as excepted.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/brown-islands-own.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-azure-devops': patch
+---
+
+Fixed **AssignedToCurrentUsersTeams** & **CreatedByCurrentUsersTeams** filter in **AzurePullRequestsPage** component.
```
Comment (awanlin) [inline .changeset/brown-islands-own.md:5]: You can just use backticks ` for this, looks a bit nicer: 

Fixed `AssignedToCurrentUsersTeams` & `CreatedByCurrentUsersTeams` filter in `AzurePullRequestsPage` component.

**Review by awanlin — APPROVED** [review]
Hi @deepan10, thanks for contibuting a fix, much appreciated. Looks good to me, left one small comment but really it's optional 😄  🚀

**Comment by awanlin** [comment]
@deepan10 can you update the description of this PR to mention the correct issue? You have #15700 when it really should be #15760. If you do that it will properly close the right issue for you. Thanks in advance!

File: .changeset/brown-islands-own.md (hunk 1 above)
Comment (deepan10) [reply .changeset/brown-islands-own.md]: Done 😄

**Review by jhaals — APPROVED** [review]
Thanks everyone

**Pull request merged**